plugins/woocommerce/includes/tracks/class-wc-site-tracking.php

```diff
@@ -187,6 +187,10 @@ public static function add_enable_tracking_function() {
 	 * Init tracking.
 	 */
 	public static function init() {
+		// Only run on admin or if it is a rest api request.
+		if ( ! is_admin() && ! WC()->is_rest_api_request() ) {
+			return;
+		}
```

Comment:
I could also move this to the `class-woocommerce` file where the actual `init` file is being triggered.

Reply:
what about the other includes?

https://github.com/woocommerce/woocommerce/pull/37796/files#diff-38b97a71209f9fae5bb245340b30743ae33dbc296b8066ea4bfffcd4d7b49c32L70-L75

Should we move that too?